frontend/src/component/AddUser.tsx

```diff
@@ -29,12 +30,24 @@ export default function AddUser(props:AddUserProps){
                     UserName:
                     <input
                         type={"text"}
-                        name={"name"}
-                        value={newUser.name}
+                        name={"username"}
+                        value={newUser.username}
                         onChange={handleOnChange}
                         placeholder={"hi, traveller"}
                     />
                 </label>
+
+                <label>
+                    Password:
+                    <input
+                        type={"password"}
+                        name={"password"}
+                        value={newUser.password}
```

Comment:
A shared state for username and pw - "[you make me happy :guitar:](https://www.youtube.com/watch?v=fFtZU2Aka80)"